index.d.ts

```diff
@@ -553,6 +553,22 @@ export declare interface TracerOptions {
      * Specifies a path to a custom blocking template json file.
      */
     blockedTemplateJson?: string,
+
+    /**
+     * Controls the automated user event tracking configuration
+     */
+    eventTracking?: {
+      /**
+       * Whether to enable automated user event tracking.
+       * @default true
+       */
+      enabled?: boolean,
+      /**
+       * Controls the automated user event tracking mode
+       * @default 'safe'
+       */
+      mode?: string
```

Comment:
We can force the accepted values, like here: https://github.com/DataDog/dd-trace-js/blob/b2420c85aae6f4c2ec2a25df0b4c586963e96fd2/index.d.ts#L411

```suggestion
      mode?: 'safe' | 'extended' | 'disabled'
```